test_node_env/expected_writer.js

```diff
@@ -0,0 +1,18 @@
+import { Script } from 'vm';
+import { createWriteStream } from 'fs';
+
+let writer2 = createWriteStream("test_node_env/expected.txt");
+const args = process.argv.slice(2);
+const expected = args[0];
+
+writer2.once('open', function(fd) {
+    let r = e.runInNewContext();
```

Comment:
Is there a reason we need to process this? The expected output seems to be the same as r when I run it locally.

Reply:
I tried but there is the problem that I don't remember. I think that it better off having a same form when print out.

Reply:
Could you check again? No point processing it if we don't need to. If not needed, we can remove `expected_writer` all together.

Reply:
Yes, when I change to this:
<img width="442" alt="Screenshot 2021-07-30 at 11 39 03 AM" src="https://user-images.githubusercontent.com/77202230/127596561-d669bf3d-ee3b-488e-b37f-e684893296c6.png">

This is the error
<img width="602" alt="Screenshot 2021-07-30 at 11 39 30 AM" src="https://user-images.githubusercontent.com/77202230/127596595-f03b5c44-72a8-4b78-8c0f-00cd25abad7d.png">

Thus, I think it better off having the same writer

Reply:
If you want to add a newline to the files, you can always add:
echo "\n" >> test_node_env/expect.txt

Reply:
Yes prof, I tried and it works for creating a new line. However, this error appears.
<img width="852" alt="Screenshot 2021-07-30 at 11 56 20 AM" src="https://user-images.githubusercontent.com/77202230/127597914-57c7d941-1dfc-4af1-ae40-2c6d11305f2a.png">

Reply:
I remember that you and me have discussed about it once

Reply:
yes I remember we solved this by applying JSON.stringify to one of the strings.

Reply:
or eval. Whatever works.

Reply:
ok let's use eval then.